gateways/http/account/get_balance.go

```diff
@@ -13,13 +13,14 @@ type AccountBalance struct {
 
 // ShowAccount godoc
 // @Summary account balance
-// @Description show the balance of a specific account
+// @Description Show the balance of a specific account
 ///@Accept  json
 // @Produce  json
 // @Success 200 {object} AccountBalance
 // @Failure 400 "Failed to decode"
 // @Failure 404 "Account not found"
 // @Failure 500 "Unexpected internal server error"
+// @Header 201 {object} AccountBalance
```

Comment:
since you're not returning 201 nor the account balance on the AccountBalance header, does make sense to keep it like this?